copilot/codeium.talon

```diff
@@ -0,0 +1,25 @@
+app: vscode
+-
+
+codeium chat: user.vscode("codeium.openChatView")
+
+codeium toggle: user.vscode("codeium.toggleEnabledForCurrentLanguage")
+
+codeium submit: key(ctrl-shift-enter)
+
+codeium command: user.vscode("codeium.openCodeiumCommand")
+
+codeium search: user.vscode("codeium.openSearchView")
+
+codeium explain: user.vscode("codeium.explainCodeBlock")
+
+codeium debug: user.vscode("codeium.explainProblem")
+
+codeium editor: user.vscode("codeium.openChatInPane")
+
+codeium jest: user.vscode("editor.action.inlineSuggest.trigger")
+codeium next: user.vscode("editor.action.inlineSuggest.showNext")
+codeium (previous | last): user.vscode("editor.action.inlineSuggest.showPrevious")
```

Comment:
@jaresty Just curious I'm wondering if some of the `pilot` commands work with `continue`. The pilot commands just use the generic editor actions,

Reply:
Yes! Some of them work exactly the same.